### PR TITLE
[DSDK-722] Added get messages by id method

### PR DESCRIPTION
**Pull request by pepcox**

### CODE REVIEW

File: library/src/main/java/ch/decent/sdk/api/rx/MessagingApi.kt [hunk 1]
```diff
@@ -172,4 +174,24 @@ class MessagingApi internal constructor(api: DCoreApi) : BaseApi(api) {
...
   }
+
+  /**
+   * Get messages by ids
+   *
+   * @param messages a list of messages ids
+   *
+   * @return list of messages
+   */
+  fun getAllByIds(messages: List<MessagingObjectId>): Single<List<Message>> =
```
Comment (fl3xman) [inline library/src/main/java/ch/decent/sdk/api/rx/MessagingApi.kt]: messages -> ids

File: library/src/main/java/ch/decent/sdk/api/rx/MessagingApi.kt [hunk 2]
```diff
@@ -172,4 +174,24 @@ class MessagingApi internal constructor(api: DCoreApi) : BaseApi(api) {
...
+          .map { it.map { Message.create(it) }.flatten() }
+
+  /**
+   * Get message by id
+   *
+   * @param message message id
+   *
+   * @return message
+   */
+  fun getById(message: MessagingObjectId): Single<Message> = getAllByIds(listOf(message)).map { it.first() }
```
Comment (fl3xman) [inline library/src/main/java/ch/decent/sdk/api/rx/MessagingApi.kt]: message -> id

File: library/src/main/java/ch/decent/sdk/net/model/request/GetMessagesObjects.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package ch.decent.sdk.net.model.request
+
+import ch.decent.sdk.model.MessageResponse
+import ch.decent.sdk.model.MessagingObjectId
+import ch.decent.sdk.net.model.ApiGroup
+import com.google.gson.reflect.TypeToken
+
+internal class GetMessagesObjects(
```
Comment (petervanderkadecent) [inline library/src/main/java/ch/decent/sdk/net/model/request/GetMessagesObjects.kt]: This should be named GetMessages

File: library/src/test/java/ch/decent/sdk/api/MessagingSuite.kt [hunk 4]
```diff
@@ -81,4 +81,14 @@ class MessagingApiTest(channel: Channel) : BaseApiTest(channel) {
           .all { it.encrypted } `should equal` true
     }
   }
+
+  @Test fun `should get messages by id`() {
```
Comment (petervanderkadecent) [inline library/src/test/java/ch/decent/sdk/api/MessagingSuite.kt]: I am not sure that order of tests is guaranteed, please add here also sending of expected messages.

Comment (petervanderkadecent) [reply library/src/test/java/ch/decent/sdk/api/MessagingSuite.kt]: Please ignore after discussion

**Pull request merged**